libs/vm/src/analyser.cpp

```diff
@@ -78,6 +80,9 @@ void Analyser::Initialise()
                       fixed32_type_);
   CreatePrimitiveType("Fixed64", TypeIndex(typeid(fixed_point::fp64_t)), true, TypeIds::Fixed64,
                       fixed64_type_);
+  CreatePrimitiveType("InitializerList", TypeIndex(typeid(InitializerListPlaceholder)), false,
```

Comment:
Move this after Null type creation, and before Void type creation

Reply:
Is there any order imposed on types creation? I thought it did not matter at all.

Reply:
Again, just for consistency.